es-runtime/src/controllers/ExtensionController.ts

```diff
@@ -109,7 +109,7 @@ class ApiImpl implements ExtensionScaffoldApi {
     }
 
     private async loadExtension(url: string) {
-        const module = await import(url)
+        const module = await import(/* @vite-ignore */ url)
```

Comment:
'vite-ignore' => Handles dynamic import errors. By adding /* @vite-ignore */, you tell Vite to treat the import as a standard runtime dynamic import without attempting to optimize it during the build process.